src/Plugin.php

```diff
@@ -55,6 +73,9 @@ public function __construct(array $config = [])
             throw new \InvalidArgumentException("Missing required configuration key 'nicks'");
         }
         $this->setNicks($config['nicks']);
+        if (!empty($config['recovery'])) {
```

Comment:
The default value of false is set in line 41. I don't understand what you're getting at.